target/linux/ramips/image/mt7621.mk

```diff
@@ -1524,6 +1524,17 @@ define Device/jcg_y2
 endef
 TARGET_DEVICES += jcg_y2
 
+define Device/jdcloud_re-cp-02
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16000k
+  DEVICE_VENDOR := JD-Cloud
+  DEVICE_MODEL := RE-CP-02
+  DEVICE_PACKAGES := kmod-mt7915-firmware kmod-sdhci-mt7620 \
+                     kmod-fs-ext4
```

Comment:
If this `kmod-fs-ext4` is unnecessary for basic use of that device, please drop it.